docs/Release_Notes.gfs.v16.1.7.txt

```diff
@@ -0,0 +1,142 @@
+GFS V16.1.7 RELEASE NOTES
+
+
+PRELUDE
+ 
+  NOAA awarded Delivery Order 4 (DO-4) of its commercial radio occultation (RO) 
+  data purchase to both Spire Global and GeoOptics on February 10, 2022.  This 
+  purchase covers 5500 occultations per day from Spire and 500 occultations per 
+  day from GeoOptics over a 10 month period with the data flow starting on 
+  March 16, 2022.
+
+  Both GeoOptics and Spire have been assimilated in operations as part of 
+  previous delivery orders.  DO-1 was awarded to both vendors, but was used 
+  for evaluation purposes only and not assimilated operationally.  DO-2 was 
+  awarded to GeoOptics and subsequently assimilated in the operational GFS/GDAS 
+  as v16.1.  DO-3 was then awarded to Spire only.  The v16.1.4 implementation 
+  turned on the assimilation of Spire data as well as turned off the assimilation 
+  of GeoOptics.  
+
+  If no changes are made to operations, we will assimilate the Spire portion of 
+  the purchase, but would not assimilate the new GeoOptics data. In order to 
+  assimilate data from both vendors, a single line change in the global_convinfo.txt 
+  fix file is required.  There are no other changes planned for this implementation.  
+  Once data flow begins on March 16th, a one-week evaluation of the new GeoOptics 
+  observations is needed before implementation.
+
+  In addition to DO-4, a small change is needed to accompany a change in the
+  observation processing.  Winds "NeXRaD VAD WINDS FROM LEVEL 2 DECODER"
+  (tank b002/xx017) will soon be included in the global observation processing.
+  Since these observations have not yet been evaluated in the GFS, this observation
+  type (uv 224) will be set to monitor mode.  This requires a single line change
+  in the global_convinfo.txt file.
```

Comment:
Prelude is from the v16.1.6 upgrade...please rewrite to be for the v16.1.7 upgrade. Can update text in your fork branch clone and commit it to your fork branch for it to show up in this PR.